src/test/resources/hudson/plugins/jira/JiraBuildActionTest/binaryCompatibility/jobs/project/builds/2/build.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<build>
+  <queueId>39</queueId>
+  <timestamp>1585749717875</timestamp>
+  <startTime>1585749717879</startTime>
+  <result>SUCCESS</result>
+  <duration>240</duration>
+  <charset>UTF-8</charset>
+  <keepLog>false</keepLog>
+  <builtOn></builtOn>
+  <workspace>/var/jenkins_home/jobs/global/workspace</workspace>
+  <hudsonVersion>2.222.1</hudsonVersion>
+  <scm class="hudson.scm.NullChangeLogParser"/>
+
+  <actions>
+    <hudson.model.CauseAction>
+      <causeBag class="linked-hash-map">
+        <entry>
+          <hudson.model.Cause_-UserIdCause>
+            <userId>admin</userId>
+          </hudson.model.Cause_-UserIdCause>
+          <int>2</int>
+        </entry>
+      </causeBag>
+    </hudson.model.CauseAction>
+
+    <hudson.plugins.jira.deprecated.DeprecatedJiraBuildAction>
+      <owner class="build" reference="../../.."/>
```

Comment:
FYI, this weird syntax is the best you can hope for if you do not use `RunAction2`. On occasion some race condition actually causes the field to serialize the entire content of the `Run`, leading to very colorful errors.